packages/docusaurus/src/webpack/plugins/WaitPlugin.ts

```diff
@@ -5,9 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import path from 'path';
-import fs from 'fs-extra';
-import waitOn from 'wait-on';
+import {stat} from 'fs/promises';
```

Comment:
Does `fs-extra` provide this? Even if not you should use `import fs from "fs/promises"` to align with our code style.